### PR TITLE
added new queries

### PR DESCRIPTION
**Pull request by Mov305**
Added all the requirements
Results

First transaction 
image

2nd transaction
image

3rd transaction
image

Last transaction
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -8,6 +8,13 @@ INSERT INTO
...
-    ('Pikachu', '2021-01-15', 0, false, 6),
-    ('Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+    ('Gabumon', '2018-11-15', 2, false, 8.00),
+    ('Pikachu', '2021-01-15', 0, false, 6.00),
+    ('Devimon', '2017-05-12', 5, true, 11.00),
+    ('Charmander', '2020-02-08', 0, false, 11.00),
+    ('Plantmon', '2021-11-15', 2, true, 5.70),
+    ('Squirtle', '1993-04-02', 3, false, 12.13),
+    ('Angemon', '2005-01-12', 1, true, 45.00),
```
Comment (Mihndim2020) [inline data.sql]: - [ ] According to the project [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), these weights have negative values, please kindly correct that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Mov305, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Mov305) [reply data.sql]: Oh thank you, I didn't realize that

**Review by Davidosky007 — APPROVED** [review]
Hi @Mov305 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All queries work as expected 👍 
* Nice looking PR description 👍  
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**